lib/sinatra/main.rb

```diff
@@ -5,7 +5,9 @@ module Sinatra
     require 'optparse'
     parser = OptionParser.new { |op|
       op.on('-p port',   'set the port (default is 4567)')                { |val| ParamsConfig[:port] = Integer(val) }
-      op.on('-s server', 'specify rack server/handler (default is puma)') { |val| ParamsConfig[:server] = val }
+      op.on('-s server', "specify rack server/handler (Of Sinatra's default servers, one with a rack handler will be adopted)") {
```

Comment:
Could we simplify this to "specify rack server/handler"? I think explaining the default is too difficult here. The list of servers could potentially be overridden right?

Reply:
Yeah i thought it would be better but considered the consistency for our UI. And the behaviour may be confusing a bit :(

Reply:
I was rethinking about it and I concluded that it should be mentioned in our README, not command line.